examples/cpp/stdio/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.16...3.27)
+
```

Comment:
Note to self: oof one of these days I have to come up with a way to make these smaller. They may be portable but they're still annoying.